sys/parsoid.yaml

```diff
@@ -58,36 +81,64 @@ paths:
           type: string
           required: false
 
-  /data-parsoid/{title}{/revision}{/tid}:
-    get:
-      summary: Retrieve the data-parsoid JSON for title & revision.
-      operationId: getDataParsoid
+  /data-parsoid/{title}/{revision}:
+    <<: *data-parsoid_title
+
```

Comment:
I do not think this is actually used ever, same as the one with no 'revision', at least it shouldn't be. But let's work on that as a part of parsoid storage simplification, not this one.